### ISSUE TITLE
[Screen Reader – BotKit]: aria-current="page" should be defined for all the pages.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#7]
**User Experience:**
Users who rely on screen readers will affect to navigate the pages if aria-current="page" is not defined.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Developer tool bar (F 12)
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in edge browser.
Step 2: Open developer tool bar (F 12)
Step 3: Navigate to “Get Started” link and verify whether narrator is narrating selected page link as Current page.

**Actual Result:**
Narrator is not narrating the Current page when user selecting “Get Started” link in header navigation.
aria-current="page" is not defined.

Ex: When focus is on “Get started “link, Narrator is narrating as “Get Started” link.

**Expected Result:**
Narrator should narrate as Current page when user selecting “Get Started” link in header navigation.
aria-current="page" should be defined.

Ex: When focus is on “Get started “link, Narrator should narrate “Get Started current page” link when link was selected.

**Note**: 
1. Same issue repros for “Docs, Support and Blog”
2. Same issue repros for all the pages which are present under “Docs”.

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}]()

[MAS1.3.1_BotKit_Pages.zip](https://github.com/howdyai/botkit-website/files/2970688/MAS1.3.1_BotKit_Pages.zip)

![MAS1 3 1_BotKit_Pages](https://user-images.githubusercontent.com/48272005/54427610-01ca6b00-4741-11e9-8ae8-eb7857027a50.png)

**Comment by benbrown** [comment]
* added to get started
* added to support
* added to all pages in the docs section

**Comment by v-gasikk** [comment]
Verified and Issue is Fixed. Hence Closing the bug.
